rllib/core/rl_module/marl_module.py

```diff
@@ -273,6 +281,77 @@ def set_state(self, state_dict: Mapping[ModuleID, Any]) -> None:
         for module_id, state in state_dict.items():
             self._rl_modules[module_id].set_state(state)
 
+    def save_state_to_dir(self, dir: Union[str, pathlib.Path]) -> str:
+        """Saves the weights of this MultiAgentRLModule to dir.
+
+        Args:
+            dir: The directory to save the checkpoint to.
```

Comment:
I think we should add the info here that the file name is determined by the module_id.
Also, the return value is not up to date!
We could return a list of the paths here if any, that would be most useful I believe.

Reply:
fixing now